src/transaction-runner.coffee

```diff
@@ -674,17 +673,17 @@ class TransactionRunner
         transaction.test = test
         return callback()
 
-  isMultipart: (requestOptions) ->
-    caseInsensitiveRequestHeaders = {}
-    for key, value of requestOptions.headers
-      caseInsensitiveRequestHeaders[key.toLowerCase()] = value
-    caseInsensitiveRequestHeaders['content-type']?.indexOf("multipart") > -1
-
-  replaceLineFeedInBody: (transaction, requestOptions) ->
-    if transaction.request['body'].indexOf('\r\n') == -1
-      transaction.request['body'] = transaction.request['body'].replace(/\n/g, '\r\n')
-      transaction.request['headers']['Content-Length'] = Buffer.byteLength(transaction.request['body'], 'utf8')
-      requestOptions.headers = transaction.request['headers']
+  isMultipart: (headers) ->
+    contentType = caseless(headers).get('Content-Type')
+    return false unless contentType
+    return contentType.indexOf('multipart') > -1
```

Comment:
Honestly, I think that change made in https://github.com/apiaryio/dredd/commit/3055aba05315ca2e4a9f68a23c124c5ed1562890 is worse than the original two returns. I think that solution from above reads a little bit better, yours has a little bit of Yoda tongue in it 🙂